build_simple_env.bat

```diff
@@ -0,0 +1,17 @@
+:: Create new network for our purposes.
+docker network create -d bridge rev-proxy-bridge-network
+
+:: Build www server image from dockerfile.
+docker build -t apache_server -f Dockerfile.www_server .
+
+:: Create www server.
+docker run -i -d --network rev-proxy-bridge-network --name www_server apache_server
+
+:: Get the www server ip. WARNING: Container must be run to get its IP.
+For /F "Tokens=*" %%I in ('docker inspect -f {{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}} www_server') Do Set WWW_SERVER_IP=%%I
```

Comment:
czemu tu jest pętla, skoro i tak tylko raz uzupełnia tą zmienną?

Reply:
To jest taki sposób do wyciągnięcia danych. Całkiem często na niego natrafiałem szukając jak wyjąć IP. Ostatecznie, z tego co rozumiem, to po prostu pętla, która wykona się raz.